config/deploy/qa.rb

```diff
@@ -2,7 +2,7 @@
 
 set :rails_env, :production
 set :bundle_without, %w[development test].join(' ')
-set :branch, 'qa'
+set :branch, 'mailerupdates'
```

Comment:
This still should be "qa".  Otherwise, when we merge it in, every time you go to deploy to libappstest in the future it will deploy the mailerupdates branch.